Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -2041,6 +2053,21 @@ private extension Purchases {
         }
     }
 
+    #if DEBUG
+    func runHealthCheckIfInForeground() {
+        self.systemInfo.isApplicationBackgrounded { isBackgrounded in
+            if !isBackgrounded {
+                self.operationDispatcher.dispatchOnWorkerThread {
+                    Task {
+                        let manager = SDKHealthManager { try await self.healthReportRequest() }
+                        await manager.logSDKHealthReportOutcome()
+                    }
+                }
+            }
+        }
+    }
```

Comment:
Do we need both the `dispatchOnWorkerThread` and `Task`?

I see there's already this function in `OperationDispatcher`:

```swift
    func dispatchOnWorkerThread(jitterableDelay delay: JitterableDelay = .none,
                                block: @escaping @Sendable () async -> Void) {
        Task.detached(priority: .background) {
            if delay.hasDelay {
                try? await Task.sleep(nanoseconds: DispatchTimeInterval(delay.random()).nanoseconds)
            }

            await block()
        }
    }
```

Reply:
Ah! Very good point, not sure how I managed to miss this, I have updated it now 👍